testfiles/DOAToPTHP.idf

```diff
@@ -157,7 +157,7 @@
 
  ! Miami Intl Ap FL USA Annual Heating Design Conditions Wind Speed=3.8m/s Wind Dir=340
  ! Coldest Month=January
- ! Miami Intl Ap FL USA Annual Heating 99.6%, MaxDB=8.7�C
+ ! Miami Intl Ap FL USA Annual Heating 99.6%, MaxDB=8.7?C
```

Comment:
Not sure if the black invalid character or the plain question mark is better here.  Actually, leaving the bad character in there is a better option because it's easier to search and replace later.  I can try to undo these changes to the files if you need help @lgu1234, since I assume you didn't manually make this change, your editor did.